src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static final int COLOR_COUNT = 5;
+    private Random random = new Random();
+    private String color;
+
+    public String getRandomColor() {
+        int colorNumber = random.nextInt(COLOR_COUNT);
+        switch (colorNumber) {
+            case 0:
+                color = "red";
+                break;

```

Comment:
```suggestion
                return "red";
```
якщо після свіча не потрібно нічого робити можеш одразу повертати значення в кейсі